runtime/lib/ttnn/program.cpp

```diff
@@ -734,6 +734,29 @@ static void run(::tt::target::ttnn::DeallocOp const *op, ::ttnn::Device &device,
   tensorPool.erase(op->in()->global_id());
 }
 
+static void run(::tt::target::ttnn::FullOp const *op, ::ttnn::Device &device,
+                ProgramTensorPool &tensorPool) {
+  const ::tt::target::DeviceRef *deviceRef = op->device();
+  assert(deviceRef->global_id() == static_cast<uint32_t>(device.id()) &&
+         "Device id mismatch");
+  ::ttnn::DataType outputDataType = getDataType(op->out());
+  auto shape = ::ttnn::Shape(::tt::tt_metal::Shape(
+      utils::toShapeFromFBShape(*op->out()->desc()->shape())));
+  float fillValue = op->fill_value();
+  // TODO(bug #272), determine correct layout by tile shape in the future
```

Comment:
For now we can probably assert that there are only 2 tile shapes and therefore infer, `[1, 1]` -> row major, or `[32, 32]` -> tiled

Reply:
Confirmed with @nsmithtt @nobradovictt that currently we don't have compiler support for dynamically generating tile shapes, will leave this hardcoded and TODO for now until we bring up the support.